src/pages/bug-bounty/vulnerabilities-types/index.tsx

```diff
@@ -72,7 +72,7 @@ const StyledLi = styled.li`
 
 const VulnerabilitiesTypes = () => {
     return (
-        <Layout type="security" margin_top={10}>
```

Comment:
this seems to be needed, or this removal is not related to your changes, can you please confirm?

Reply:
Same here.